### ISSUE TITLE
dox() on http

### ISSUE THREAD

**Issue by jbolila** [thlorenz/replpad#24]
the Node documentation is just available over https now!?

```javascript
 , rootUrl = format('http://nodejs.org/docs/%s/api', process.version)
```

now is giving me the error `ERR! [Error: Documentation at [http://nodejs.org/docs/v4.2.4/api/fs.json] not found.]`, (but https://nodejs.org/docs/v4.2.4/api/fs.json exists).

**Comment by thlorenz** [comment]
Yeah we'd have to update the url it uses for new (or all?) node versions.
I don't have time to take care of this, but would appreciate a PR that fixes that, look [around here](https://github.com/thlorenz/replpad/blob/0cfe71405c82dcc1066e98c2ce5c818f59c2958d/lib/dox/core/print.js#L12-L19)

**Comment by jbolila** [comment]
done! https://github.com/thlorenz/replpad/pull/25

**Comment by davidhq** [comment]
I also sent a pull request before checking the issues, hopefuly one of them will be merged...

By the way I would like to run replpad with `--use_strict` option, cannot find where to add this in the code... so that replpad starts node with this option ? Any hints? thank you

**Comment by thlorenz** [comment]
@davidhq for `--use_strict` just run 
```
node --use_strict `which replpad`
```
but then you'll actually get an error since `mkdirp` uses octal literals, so not a good idea anyways.
